style.css

```diff
@@ -39,15 +48,27 @@ input {
     inline-size: 100%;
     padding: 10px;
     border: 1px solid #292929;
+    border-radius: 10px;
 }
 
 .shape {
     scale: 5;
-    margin-block-start: 50px;
+    padding: 30px;
+    margin-block-start: 30px;
 }
 
 a {
     position: absolute;
     inset-block-start: 0;
     inset-inline-end: 0;
 }
+
+.bottom-image {
+    width: 100%;            
+    height: auto;           
+    display: block;         
+    object-fit: cover;
+    position: absolute;      
+    bottom: 0;                 
+    left: 0;  
```

Comment:
change it with CSS logical properties
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values